docs/api/request.md

```diff
@@ -105,7 +105,7 @@ const ct = this.request.type;
   Get request charset when present, or `undefined`:
 
 ```js
-this.request.charset
+this.request.charset;
```

Comment:
Well, there is no nice solution. It is discussed on the plugin page: eslint/eslint-plugin-markdown#46.

These are possible solutions:

* It could be left unhighlighted.
* It could be marked as a different but similar language, e.g. Typescript.
* It could be disabled with `// eslint-disable-line` or `// eslint-disable-next-line`.
* Semicolons could be optional in Markdown.
* Some files could be unlinted.

Is there one that would be acceptable?